aries-site/src/pages/templates/page-layouts.js

```diff
@@ -95,7 +101,98 @@ const PageLayouts = () => {
         </Subsection>
       </ContentSection>
       <ContentSection>
-        <Subsection name="Sidebar, header, and footer">
+        <Subsection name="Guidance">
+          <SubsectionText>
+            The when, how, and why to use page layouts.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="About Page Layouts" level={3}>
+          <SubsectionText>
+            Choosing a page layout is an important first step in designing an
+            application or website. Use of these layouts will help to maintain
+            consistency across your design. They also provide visual guidance
+            for how your design behaves at different screen sizes.
+          </SubsectionText>
+          <Text weight="bold">Common use cases include:</Text>
+          <BulletedList
+            items={[
+              'Creating a dashboard with global navigation',
+              'Providing a dialog for viewing or updating something',
+              'Displaying a page in a website',
+            ]}
+          />
+          <Text weight="bold">Components used within a Page Layout:</Text>
+          <SubsectionText>
+            Page Layouts are built using the{' '}
+            <Anchor label="Header" href={nameToPath('Header')} /> and{' '}
+            <Anchor label="Main" href={nameToPath('Main')} /> components, with
+            optional{' '}
+            <Anchor
+              label="Global Sidebar"
+              href={nameToPath('Global Sidebar')}
+            />{' '}
+            and <Anchor label="Footer" href={nameToPath('Footer')} />{' '}
+            components. For guidance on page layout content, refer to the HPE
+            Design System website pages for the components that make up your
+            Page Layout.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Usage" level={3}>
+          <SubsectionText>
+            Page Layouts provide the basic structure for a variety of page
+            types.
+          </SubsectionText>
+          <BulletedList
+            items={[
+              `Page-level vertical scrolling is supported, but horizontal 
+              scrolling is not (and should be avoided, when possible).`,
+              <Text>
+                Can be used with various{' '}
+                <Anchor
+                  label="Navigation"
+                  href={nameToPath(`Navigation
+                `)}
+                />{' '}
```

Comment:
This link is not currently working. This should correct it.
```suggestion
                <Anchor
                  label="Navigation"
                  href={nameToPath('Navigation')}
                />{' '}
```